package.json

```diff
@@ -165,22 +163,19 @@
     "electron-prebuilt": "brave/electron-prebuilt",
     "empty-port": "0.0.2",
     "enzyme": "^2.9.1",
-    "eslint-plugin-flowtype": "^2.40.1",
-    "flow-bin": "^0.53.1",
     "font-awesome-webpack": "~0.0.4",
     "git-rev-sync": "^1.8.0",
-    "gulp": "^3.9.0",
     "istanbul": "^1.1.0-alpha.1",
     "joi": "^10.2.2",
     "jsdom": "~11.2.0",
     "jsdom-global": "~3.0.2",
     "json-loader": "^0.5.4",
     "jsonfile": "^2.2.3",
-    "less": "^2.5.3",
+    "less": "^3.0.4",
     "less-loader": "^2.2.1",
     "lolex": "~1.3.2",
     "mkdirp": "^0.5.1",
-    "mocha": "^5.2.0",
+    "mocha": "^2.5.3",
```

Comment:
this regresses https://github.com/brave/browser-laptop/pull/15165 which causes the `1 critical vulnerability`